Ports/RetroArch/patches/ReadMe.md

```diff
@@ -31,6 +31,7 @@ Set default options for SerenityOS
 
 Set libretro cores path to `/usr/lib/libretro`
 Set video and audio driver to sdl2
+Set video scale to 2
```

Comment:
Please also add this line to the subject of the corresponding patch.

Reply:
I changed some default configuration which is part of `0006-Set-default-options-for-SerenityOS.patch`

Reply:
Usually you edit the patch and regenerate the ReadMe. The text in the subject of the patch should match what is in the ReadMe, so in patch 0006 line 8 should read `Set video scale to 2`, just like the ReadMe.